beacon-chain/db/kv/finalized_block_roots_test.go

```diff
@@ -0,0 +1,93 @@
+package kv
+
+import (
+	"context"
+	"testing"
+
+	"github.com/prysmaticlabs/go-ssz"
+	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+	ethpb "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
+	"github.com/prysmaticlabs/prysm/shared/featureconfig"
+)
+
+var genesisBlockRoot = bytesutil.ToBytes32([]byte{'G', 'E', 'N', 'E', 'S', 'I', 'S'})
+
+func init() {
+	fc := featureconfig.Get()
+	fc.EnableFinalizedBlockRootIndex = true
+	featureconfig.Init(fc)
+}
+
+func TestStore_IsFinalizedBlock(t *testing.T) {
```

Comment:
Is it possible to cover a few more cases ? ex: Across multiple epochs, with skipped slots at epoch boundaries,etc

Reply:
Hmmm, I'll think about overnight. There isn't any different code path for mulitple epochs with skipped slots. Epoch boundries aren't relevant here. As long as the finalized chain is a chain then this should work.